src/docfx/build/toc/BuildTableOfContents.cs

```diff
@@ -44,9 +61,44 @@ private static Task BuildTocMap(Context context, Document fileToBuild, TableOfCo
             return Task.CompletedTask;
         }
 
-        private static (TableOfContentsModel tocModel, List<Document> referencedDocument, List<Document> referencedTocs) Load(Document fileToBuild)
+        private static (List<TableOfContentsModel> tocModel, List<Document> referencedDocument, List<Document> referencedTocs) Load(Document fileToBuild)
         {
-            throw new NotImplementedException();
+            if (fileToBuild == null)
```

Comment:
this should never be null

Reply:
removed

---
In reply to: [185712505](https://github.com/dotnet/docfx/pull/2717#discussion_r185712505) [](ancestors = 185712505)